### DISCUSSION TITLE
Looking for the source of the "female_03.wav" voice file

### DISCUSSION THREAD

**Discussion by ngphuchoangnam** [erew123/alltalk_tts#346] [Q&A] [answered]
Does anyone knows where that voice comes from? The file named "female_03.wav" in the "voices" folder.
I want to further fine tune on that voice because it sounds great for audiobook, however for some reason it's pretty unstable. I suspect whisper did not transcribe the samples properly so I'll do manual transcribe and cleaning data to see if it's better.
Just need to know where it's from, like what movie or audiobook...
🙏 I pray that erew saw this post cause that would save a lot of searching.

**Comment by erew123** [comment, marked as answer]
This I think

https://freesound.org/people/Sample_Me/sounds/623171/

Thanks
